test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -72,7 +73,7 @@ var _ = SIGDescribe("CustomResourcePublishOpenAPI [Privileged:ClusterAdmin]", fu
 		meta := fmt.Sprintf(metaPattern, crd.Crd.Spec.Names.Kind, crd.Crd.Spec.Group, crd.Crd.Spec.Versions[0].Name, "test-foo")
 		ns := fmt.Sprintf("--namespace=%v", f.Namespace.Name)
 
-		ginkgo.By("client-side validation (kubectl create and apply) allows request with known and required properties")
+		ginkgo.By("server-side validation (kubectl create and apply) allows request with known and required properties")
```

Comment:
making the `ginkgo.By` descriptions neutral whether they are using server or client-side validation is fine, but don't change the conformance description to refer to a beta feature. If anything, make the description more neutral, since the test no longer dictates whether server-side or client-side validation is used.

Reply:
modified %s/server-side validation/kubectl validation/